packages/request-client.js/test/index.test.ts

```diff
@@ -1528,14 +1536,18 @@ describe('index', () => {
         '0x38c44820c37d31fbfe3fcee9d4bcf1b887d3f90fb67d62d924af03b065a80ced',
       );
       jest.useRealTimers();
-    }, 20000);
+    });
 
     it('can get the balance on a skipped payment detection request', async () => {
       jest.useFakeTimers('modern');
 
-      jest
-        .spyOn(ethers.providers, 'EtherscanProvider')
-        .mockImplementation(() => new EtherscanProviderMockup() as any);
+      const etherscanMock = new EtherscanProviderMock();
+      ethers.providers.EtherscanProvider.prototype.getHistory = jest
+        .fn()
+        .mockImplementation(etherscanMock.getHistory);
+      ethers.providers.EtherscanProvider.prototype.getNetwork = jest
+        .fn()
+        .mockImplementation(etherscanMock.getNetwork);
```

Comment:
it's no longer possible to do the `spyOn` method so I changed the method, but the result is the same